gdx/src/com/badlogic/gdx/scenes/scene2d/utils/TiledDrawable.java

```diff
@@ -78,5 +83,13 @@ public void draw (Batch batch, float x, float y, float width, float height) {
 				x += regionWidth;
 			}
 		}
+		batch.setColor(batchColor);
+	}
+
+	/** Creates a new drawable that renders the same as this drawable tinted the specified color. */
+	public TiledDrawable tintTiled (Color tint) {
+		TiledDrawable drawable = new TiledDrawable(this);
+		drawable.tint = tint.toFloatBits();
+		return drawable;

```

Comment:
Perhaps the tinting functionality should be moved up to the superclass TextureRegionDrawable, and TextureRegionDrawable.tint() can return TextureRegionDrawable instead of SpriteDrawable. Then this could override that method for better clarity. 

I didn't do that because I don't know the design decision made before: why TextureRegionDrawable doesn't support tinting and defers to SpriteDrawable. Maybe in the past, there was no batch.getPackedColor()  or batch.setColor(float), so holding a Color object was deemed to heavy?